toolchest_client/tools/fastqc.py

```diff
@@ -0,0 +1,27 @@
+"""
+toolchest_client.tools.fastqc
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This is the FastQC implementation of the Tool class.
+"""
+from toolchest_client.files import OutputType
+from toolchest_client.files.s3 import path_is_s3_uri
+
+from . import Tool
+
+
+class FastQC(Tool):
+    """
+    The FastQC implementation of the Tool class.
+    """
+    def __init__(self, tool_args, inputs, output_path, **kwargs):
+        super().__init__(
+            tool_name="fastqc",
+            tool_version="0.11.9",  # todo: allow salmon version to be set by the user
```

Comment:
Nit: `salmon` -> `fastqc`. Although we can probably also start dropping these comments / setting a default tool_version but adding as a kwarg